tests/phpunit/tests/interactivity-api/wpInteractivityAPIFunctions.php

```diff
@@ -390,4 +390,31 @@ public function test_wp_interactivity_data_wp_context_with_json_flags() {
 		$this->assertEquals( 'data-wp-context=\'{"quot":"\u0022baz\u0022"}\'', wp_interactivity_data_wp_context( array( 'quot' => '"baz"' ) ) );
 		$this->assertEquals( 'data-wp-context=\'{"amp":"T\u0026T"}\'', wp_interactivity_data_wp_context( array( 'amp' => 'T&T' ) ) );
 	}
+
+	/**
+	 * Tests that directives processing of tags that don't visit closer tag work.
+	 *
+	 * @ticket 60746
+	 *
+	 * @covers ::wp_interactivity_process_directives_of_interactive_blocks
+	 */
+	public function test_process_directives_in_tags_that_dont_visit_closer_tag() {
```

Comment:
Could this test be expanded (or another one added) which test that the absence of the new `if` condition does indeed cause breaking the whole server-side rendering processing as described?

Reply:
Not sure if I understand what you mean. The existing test already checks that the SSR doesn't break when there is an iframe. The same test was failing without the final solution.

By the way, I've reverted the `if` condition because it is solved by another PR as explained [here](https://github.com/WordPress/wordpress-develop/pull/6247#issuecomment-1989598180).